### DISCUSSION TITLE
Darstellungsvarianten - User Interface - Gruppierung

### DISCUSSION THREAD

**Discussion by Andreas-Lampert** [e-netze/webgis-community#104] [Q&A]
Hallo Jürgen!

Es kam noch eine Frage zum TOC auf. Ich hoffe meine Beschreibung ist verständlich.

Es sollen im Container Hochwasser in Summe 9 Rasterlayer mittels OptionBox angezeigt werden (Wasserspiegel, Wassertiefen und Fließgeschwindigkeiten). Die OptionBox wird verwendet um die gleichzeitige Anzeige mehrerer Raster zu unterbinden (Raster sind Deckungsgleich, Last am Server gering halten).

![image](https://github.com/gis-eni/webgis-forum/assets/122986367/9029afd3-5be8-431a-b7e7-09c93f068496)

Zusätzlich kam der Wunsch, diese 9 Raster je nach Thema in in Summe drei Gruppen zu untergliedern. 
Damit ist es dann aber im TOC möglich, in jeder Gruppe jeweils einen Rasterlayer einzuschalten.

![image](https://github.com/gis-eni/webgis-forum/assets/122986367/cbaf354b-6dce-43a8-a02c-efab83143124)

Gibt es eine Möglichkeit bei der Konfiguration einer Gruppierung das gleichzeitige Einschalten mehrerer Raster durch den User zu verhindern bzw. mit Klick auf die OptionBox die anderen Rasterlayer automatisch auszuschalten?
Mein derzeitiger (unsauberer) Workaround ist die Verwendung von Buttons, denn damit wird immer nur der jeweils geklickte Layer angezeigt. Aber damit sieht der User nicht mehr, welcher Layer gerade aktiv ist. 

 
![image](https://github.com/gis-eni/webgis-forum/assets/122986367/ad09c50c-1f19-4706-a6c7-3254da95a021)

Und noch eine Frage zum Feld Gruppierung. Wenn ich dort einen Wert mit einem Schrägstrich (m/s) einfüge, dann wird im TOC die Gruppe zweizeilig. Kann das verhindert werden?

![image](https://github.com/gis-eni/webgis-forum/assets/122986367/73bb5bfc-9495-47e9-8b29-6b7d587be53d)

Danke und ein schönes Wochenende!

Lg, Andi

**Comment by gis-eni** [comment]
Hallo Andi,

Option Boxes funktionieren per Definition immer nur innerhalb einer Gruppe. Das heißt, wenn du sie in mehrere Gruppen aufteilst, verhindern sie nur innerhalb dieser Gruppe, dass nur eine Thema gleichzeitig eingeschalten wird. (wie du ja eh schon richtig erkannt hast)

Deine "unsaubere" Lösung mit den Buttons funktioniert. Wenn nichts anders im Dienst drinnen ist, kann man das so machen. Der Anwender sieht dann halt nicht mehr auf einen Blick, was gerade eingeschalten ist...

Das mit dem "/" im Namen schau ich mir an. Eventuell gibt es dafür eine Lösung

**Comment by gis-eni** [comment]
Noch einmal zu (m/s):

das kann ich bei mir leider nicht nachvollziehen. Da schaut das sowohl mit `/` als auch mit `\` ganz normal aus:

![image](https://github.com/gis-eni/webgis-forum/assets/62870568/775acf51-d9c4-4c05-b597-770d51d54d35)

Was machst du da genau?

**Comment by Andreas-Lampert** [comment]
Hallo Jürgen!

Danke für deine Antwort!

Das Problem mit dem "/" tritt auf, wenn ich etwas unter User Interface - Gruppierung hineinschreibe.

![image](https://github.com/gis-eni/webgis-forum/assets/122986367/9313cccb-873b-4623-9f1c-ed31b63f50dc)

Schon im CMS bei der Darstellungsvariante wird es anders dargestellt (m >> s)

![image](https://github.com/gis-eni/webgis-forum/assets/122986367/85001d88-bdda-4e8d-b50f-084b0c7d4325)

Und im WebGIS habe ich es dann zweizeilig.

![image](https://github.com/gis-eni/webgis-forum/assets/122986367/be5d8cec-6e5e-4819-82a9-d0424c834fa4)

Ich bin bei mir lokal auf Version 5.23.2802.
Danke!

Lg, Andi

**Comment by gis-eni** [comment]
Das Problem ist hier, dass ``/`` das Trennzeichen ist um mehrere Ebenen anzugeben.
Ab Build 5.23.3901 gibt es dafür einen Lösung:

![image](https://github.com/gis-eni/webgis-forum/assets/62870568/3f602a1d-2308-44ba-b722-b91bacb67976)

Die Eingabe kann dann zB folgendermaßen lauten:

``Level 1/Level 2 (km\/h)``

oder in deinem Beispiel:

``Fließgeschwindigkeit (m\/s)``
